src/interpolate/interpolate.d.ts

```diff
@@ -0,0 +1,9 @@
+import type { Color, Find, Mode, OverridesObject } from '../common';
+
+type ColorPosition = [Color | string, number];
+
+declare function interpolate<M extends Mode>(
```

Comment:
I'm not 100% sure that this type declaration is 100% corresponds to the `interpolate` function.

Looks like not all possible usages of the function can be found in tests.